clients/admin-ui/cypress/e2e/routes.cy.ts

```diff
@@ -30,13 +30,23 @@ describe("Routes", () => {
       cy.visit("/datastore-connection");
       cy.wait("@getConnectors");
       cy.getByTestId("connection-grid");
+      cy.visit("/privacy-requests/configure");
+      cy.getByTestId("privacy-requests-configure");
+    });
+
+    // This doesn't work right now due needing a fix for `exact` in the `nav-config` helpers
+    // This is the same issue as https://github.com/ethyca/fides/issues/2731
```

Comment:
this should be fixable, but I didn't have time to get to it

related to https://github.com/ethyca/fides/issues/2731

Reply:
resolving to get this merged and we can come back and fix this later -